projects/core/utils/miscellaneous/provide-taiga.ts

```diff
@@ -0,0 +1,78 @@
+import {DOCUMENT} from '@angular/common';
+import {
+    effect,
+    type EnvironmentProviders,
+    inject,
+    InjectionToken,
+    provideAppInitializer,
+    type Provider,
+} from '@angular/core';
+import {REMOVE_STYLES_ON_COMPONENT_DESTROY} from '@angular/platform-browser';
+import {tuiIsPresent} from '@taiga-ui/cdk/utils/miscellaneous';
+import {TUI_DARK_MODE} from '@taiga-ui/core/tokens';
+import {provideEventPlugins} from '@taiga-ui/event-plugins';
+
+import {tuiEnableFontScaling} from './font-scaling';
+
+interface ExperimentalAPIs {
+    readonly all: boolean;
+}
+
+export interface TuiOptions {
+    readonly mode?: 'dark' | 'light';
+    readonly apis: Partial<ExperimentalAPIs> | 'stable';
+    readonly fontScaling: boolean;
+    readonly scrollbars: 'custom' | 'native';
+}
+
+const DEFAULT: TuiOptions = {
+    apis: 'stable',
+    fontScaling: true,
+    scrollbars: 'custom',
+};
+
+export const TUI_OPTIONS = new InjectionToken<TuiOptions>('[TUI_OPTIONS]');
+
+// eslint-disable-next-line @typescript-eslint/naming-convention
+export function provideTaiga(
```

Comment:
Add it here to fix Cypress component tests

https://github.com/taiga-family/taiga-ui/blob/16158e194bcf3bd2ae8317aacdaa1db6093fc412/projects/demo-cypress/src/support/component.ts#L20-L27